src/valkey-benchmark.c

```diff
@@ -183,13 +183,13 @@ typedef struct clusterNode {
     int *updated_slots;      /* Used by updateClusterSlotsConfiguration */
     int updated_slots_count; /* Used by updateClusterSlotsConfiguration */
     int replicas_count;
-    struct serverConfig *redis_config;
+    struct benchmarkConfig *benchmark_config;
 } clusterNode;
 
-typedef struct serverConfig {
+typedef struct benchmarkConfig {
     sds save;
     sds appendonly;
-} serverConfig;
+} benchmarkConfig;
```

Comment:
Wait a minute. Actually `save` and `appendonly` sounds like valkey server configs. :thinking:

Is this actually valkey server configs, the benchmark tool gets this info from the valkey server?

There is another struct which is just called `static struct config { ... }` and it has all the benchmark configs like pipeline, datasize, etc. from the command line options.

Maybe the first commit was correct? :man_facepalming:

Reply:
Looks like these are from the server itself: https://github.com/valkey-io/valkey/blob/73696bf6e2cf754acc3ec24eaf9ca6b879bfc5d7/src/valkey-benchmark.c#L310-L311

`serverConfig` makes more sense to me with this context